README.md

```diff
@@ -46,102 +68,203 @@ The use cases in this repository follow a flow similar to this diagram:
+  c. If you examine the output, you will see that the change to the JSON object caused Terraform to detect a delta between the state stored in S3 (with the old JSON code) and what is present in the new code:
+  
+```bash
+Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the
+following symbols:
+  ~ update in-place
+
+Terraform will perform the following actions:
 
-5. Click "Create a branch" and choose "Checkout Locally" for GitHub to create a development branch and PingOne environment on your behalf. `cd` into the local root of this repository and enter the steps presented from "Checkout Locally". 
+  # davinci_flow.registration_flow will be updated in-place
+  ~ resource "davinci_flow" "registration_flow" {
+      ~ flow_configuration_json = (sensitive value)
+      ~ flow_export_json        = (sensitive value)
+      ~ flow_json               = (sensitive value)
+      id                      = "a6d551f1d7aa2612f2bf6c371b0026e1"
+        name                    = "PingOne DaVinci Registration Example"
+        # (4 unchanged attributes hidden)
 
-6. Run the deploy command to initialize the application
+        # (3 unchanged blocks hidden)
+    }
+
+Plan: 0 to add, 1 to change, 0 to destroy.
+```
+
+  d. Applying the change will update and 'redeploy' the flow (even though it is exactly the same as what is there, the update to the file creates a delta for Terraform to resolve) and afterward, the state will reflect the new configuration. Type `yes` to apply the changes.
+
+  e. Run the script again to confirm that the state matches the configuration:
 
 ```bash
-./scripts/local_feature_deploy.sh
+No changes. Your infrastructure matches the configuration.
+Terraform has compared your real infrastructure against your configuration and found no differences, so no changes are
+needed.
+
+Apply complete! Resources: 0 added, 0 changed, 0 destroyed.
+```
+
+13. If you want to go one step further, you can modify the HTML code that is placed in the nginx image. To do so, you can modify `./terraform/sample-app/index.html` in some manner.  When the script is ran again, it will detect the change to the file, rebuild the Docker image, and launch a replacement container for the UI.
+
+14. Before committing and pushing the changes, run a devcheck against the code to ensure the formatting and syntax are correct, ignoring any warnings or informational messages:
+
+```bash
+make devcheck
+```
+
+15. Commit and push the changes to the repository:
+
+```bash
+git add .
+git commit -m "Adding password requirements to registration page"
+git push
+```
+
+16. The push will fire a pipeline that runs the same checks as you did locally. As it running on a local development branch, no deployment will occur.
```

Comment:
```suggestion
16. The push will fire a pipeline that runs the same checks as you did locally. However, since this is a development  branch, Terraform deployment will be skipped.
```